cmd/image-autobumper/main.go

```diff
@@ -0,0 +1,624 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/kyma-project/test-infra/cmd/image-autobumper/bumper"
+	"github.com/kyma-project/test-infra/cmd/image-autobumper/imagebumper"
+	"github.com/sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+	"golang.org/x/oauth2/google"
+	"gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// AutoBumpConfig contains path to AutoBump config file
+	AutoBumpConfig string
+
+	// GitHubToken contains path to GitHub token
+	GitHubToken string
+
+	// tagRegexp is the regular expression to match a tag.
+	tagRegexp = regexp.MustCompile("v[0-9]{8}-[a-f0-9]{6,9}")
+
+	// imageMatcher is the regular expression to match an image.
+	imageMatcher = regexp.MustCompile(`(?s)^.+image:(.+):(v[a-zA-Z0-9_.-]+)`)
```

Comment:
Comment could be handy for those who do not use such regexp features on daily basis. This is not simple expression.